move/governance/sources/governance/governance.move

```diff
@@ -107,4 +107,321 @@ module governance::governance {
     fun is_cap_new(cap: &UpgradeCap) {
         assert!(package::version(cap) == 1, ENotNewPackage);
     }
+
+    // -----
+    // Tests
+    // -----
+
+    #[test_only]
+    use std::ascii;
+    #[test_only]
+    use sui::test_scenario;
+    #[test_only]
+    use sui::test_utils;
+
+    #[test]
+    fun test_new() {
+        let trusted_source_chain = ascii::string(b"ethereum");
```

Comment:
trusted source chain is meant to be "Axelar" but it doesn't really matter for testing.